Framework/Algorithms/src/DetectorEfficiencyCorUser.cpp

```diff
@@ -113,8 +113,10 @@ void DetectorEfficiencyCorUser::exec() {
  * @param eOut corrected spectrum errors
  */
 void DetectorEfficiencyCorUser::applyDetEfficiency(
-    const size_t numberOfChannels, const MantidVec &yIn, const MantidVec &eIn,
-    const MantidVec &effVec, MantidVec &yOut, MantidVec &eOut) {
+    const size_t numberOfChannels, const Mantid::HistogramData::HistogramY &yIn,
+    const Mantid::HistogramData::HistogramE &eIn, const MantidVec &effVec,
+    Mantid::HistogramData::HistogramY &yOut,
+    Mantid::HistogramData::HistogramE &eOut) {
```

Comment:
Personally, I would refactor this function to have a signature like this: `Histogram applyXYZ(const Histogram &histogram)`. THat would also make the call site look cleaner, I think.